{{ cookiecutter.project_slug }}/templates/healthz.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<html>
+  <head><title>Healthcheck</title></head>
+  <body>ok</body>
+</html>
```

Comment:
I think this should be ``{{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/templates/{{ cookiecutter.project_module }}/healthz.html``. Then the addition to ``TEMPLATES['DIRS']`` is not needed.

More generally, I don't think the /healthz endpoint needs the overhead of running the template engine. How about just something simple like:

```python
# file: {{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/views.py
#    or {{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/urls.py

from django.http import HttpResponse

def healthz(request):
    HttpResponse('ok', content_type="text/plain")
```